_emails/holly-springs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Holly Springs, NC
+permalink: "/holly-springs"
+name: Letter to Council Members, and Mayor
+city: Holly Springs
+state: NC
+recipients:
+- dick.sears@hollyspringsnc.us
+- dan.berry@hollyspringsnc.us
+- christine.kelly@hollyspringsnc.us
+- shaun.mcgrath@hollyspringsnc.us
+- peter.villadsen@Hollyspringsnc.us
+- aaron.wolff@hollyspringsnc.us
+subject: Decrease funding for the Holly Springs police department
+body: |-
+  Dear Mayor Sears and Town Council,
+
+My name is [NAME] and I am a resident of Holly Springs. I urge you to amend the proposed budget for 2021. I would like to redirect money away from Holly Springs Police Department and into social service programs that will benefit public health and our own community directly, especially our community of color. The proposed budget for 2021 suggests that the largest fraction of the town's budget should go to policing ($7,861,585), which is nearly 30% larger than even the second largest allocation of funds, parks and recreation ($6,082,691). The recommended budget for FY20-21 suggests that 25% of the general fund changes should go to a "Safe and Friendly" priority area while only 2% should go to "Economic Prosperity and Diversity" and an "Engaged, Healthy, and Active Community," respectively. You must consider that economic prosperity and diversity, as well as public health in general, have so much more to do with the safety of a community than does a police force, especially as we see our nation reel and ache over the unjust murder of George Floyd and countless other black Americans at the hands of police forces, not to mention the police brutality that has ensued as a direct result of protests against police brutality. Holly Springs already applauds itself for being the "#1 Safest City in North Carolina." We do not need to spend so much on policing.
```

Comment:
Please add 2 spaces before each line below `body`. Missing indents will break the template. Thank you

Reply:
Done.